azerothcore/azerothcore-wotlk
# fix(DB/BlackwingLair): Supression Room trash mobs should not respawn …

…once Broodlord Lashlayer is killed.

Fixes #11997



## Issues Addressed:

- Closes #11997
- Closes https://github.com/chromiecraft/chromiecraft/issues/3593

## Tests Performed:

- Not tested.

## How to Test the Changes:

`.go xyz -7646 -995 441 469`
Clear the whole room and kill the boss
Wait 10 mins
See no trash mobs respawning anymore


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
